scripts/check_codebase_language.py

```
@@ -0,0 +1,177 @@
+            elif in_docstring:
+                content = line.strip()
+                if content:
+                    comments.append((content, i))
+
+    except Exception as e:
+        print(f"Error reading {file_path}: {e}")
+
+    return comments
+
+
+def check_file_with_languagetool(file_path: str, tool) -> list:
```

Ruff (0.12.2)

Missing type annotation for function argument `tool`

(ANN001)